pages/inspector/inspector-issues-beta.mdx

```diff
@@ -0,0 +1,61 @@
+# Inspector Issues View beta
+
+*Information and disclaimers for the Inspector issues view closed beta*
+
+## What is new in this beta?
+
+In this beta we're introducing a new infrastructure that generates the issues. In addition we've added alerts for saved views.
+
+### New infrastructure
+
+#### What to look out for in this beta
+
+We've built a new infrastructure for generating issues, that allows for more flexibility and speed. We are still tweaking the issue generation and the issue details so in the beginning you might encounter:
+- Issues with no issue event volume
+- Edge cases where an issue is displayed that may not be an actual issue
+-
```

Comment:
@bjornj12 @bragur and @hugihlynsson can you please help with the disclaimers here?

Reply:
I don't remember anything else!